### DISCUSSION TITLE
Capture of cities

### DISCUSSION THREAD

**Discussion by 13RDPALEON** [Vdauphin/HeartsAndMinds#1519] [Question and answer]
Hello V since version 1.23.3 the cities capture from the rest not after restarting the server an idea?

**Comment by Vdauphin** [comment]
Hello,
Check your .rpt you may have some error
Cheers

**Comment by 13RDPALEON** [comment]
There is no error in the rtp, and the rtp is not used to know if there are script errors so it does not really help me by saying "look at the rtp". This comes from a problem in your script. Certainly in the fnc City folder, but where?

Cheers

**Reply by Vdauphin** [reply]
> nd the rtp is not used to know if there are script errors 

Yes it does.

> This comes from a problem in your script.

Then you must create an issue not a discussion. Discussion is not for bug reports as there are lacking of any context
See this https://github.com/Vdauphin/HeartsAndMinds/discussions/1277#discussioncomment-1942566

**Reply by 13RDPALEON** [reply]
It’s not gonna help me find the problem.
In this post the person asks to create a specific thing, but I’m just pointing out that there is a problem in the scripts. Then do what you want, I don’t want to create a problem, I just want to let you know that there is an error in one of the scripts, then it’s up to you to fix it.
